doc/source/openfisca_serve.rst

```diff
@@ -11,7 +11,13 @@ openfisca serve
 Additional arguments
 --------------------
 
-``openfisca serve`` uses ``gunicorn`` under the hood. In addition to the arguments listed above, you can use any ``gunicorn`` arguments when running ``openfisca serve`` (e.g. ``--reload``, ``--workers``, ``--timeout``).
+``openfisca serve`` uses ``gunicorn`` under the hood. In addition to the arguments listed above, you can use any ``gunicorn`` arguments when running ``openfisca serve`` (e.g. ``--reload``, ``--workers``, ``--timeout``, ``--bind``).
+See:
```

Comment:
Could this all fit in one line ? e.g. `(see XXX for more information)`

Reply:
It don't see any impediment to this being merged as it is, I'd leave it to @sandcha to decide.

Reply:
I suggest do it in another PR as it's an improvement and not blocking to merge Joãn commits.  

> Besides, this is the list arguments that we tested on command line and in configuration file. 
If we replace this list with a link to [gunicorn documentation](http://docs.gunicorn.org), we should find a link to the right gunicorn revision and I didn't find it.